cluster/defaults.go

```diff
@@ -807,13 +823,10 @@ func (c *Cluster) setClusterNetworkDefaults() {
 		networkPluginConfigDefaultsMap[AciEncapType] = c.Network.AciNetworkProvider.EncapType
 		networkPluginConfigDefaultsMap[AciMcastRangeStart] = c.Network.AciNetworkProvider.McastRangeStart
 		networkPluginConfigDefaultsMap[AciMcastRangeEnd] = c.Network.AciNetworkProvider.McastRangeEnd
-		networkPluginConfigDefaultsMap[AciNodeSubnet] = c.Network.AciNetworkProvider.NodeSubnet
```

Comment:
We don't remove variables this way cuz it causes upgrades on RKE reconcile. But looks like the net result in template might be the same here because we're changing the logic to assign field from struct field instead. @LavanyaVaddavalli what is the reasoning behind moving it from Network.Options to a first class field under struct now? Users would be able to pass it both ways so I'm curious on why we're moving certain fields now.  

Same for DynamicExternalSubnet and StaticExternalSubnet.